apps/web/src/components/DragLayer.tsx

```diff
@@ -0,0 +1,161 @@
+import { CSSProperties, useEffect } from 'react'
+import { useDragLayer, XYCoord } from 'react-dnd'
+import IconSelector from './IconSelector'
+import { ElementType } from './v2Editor'
```

Comment:
I think this is what's causing the ssr error by the way.

When you import `./v2Editor` you end up importing `./customBlocks/python` which imports `./PythonOutput`.

Rule of thumb is, do not import stuff from `v2Editor` with `ssr`, a lot of stuff from that folder do not work with `ssr`.

You should probably just move `ElementType` to it's own file, maybe we can introduce a `utils/dragndrop.ts` file.